include/swift/AST/DiagnosticsSema.def

```diff
@@ -872,6 +872,31 @@ ERROR(need_hermetic_seal_to_import_module,none,
       "current compilation does not have -experimental-hermetic-seal-at-link",
       (Identifier))
 
+ERROR(modularization_issue_moved,Fatal,
```

Comment:
maybe modularization_issue_decl_moved to make it more explicit?

Reply:
Added `decl_` to all of them and the related diagnostics.